### PR TITLE
Copy csc/mcs GC settings to msbuild script

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Comment by marek-safar** [comment]
This needs to go to 2017-02 too

**Review by radical — APPROVED** [review]

**Comment by radical** [comment]
BTW, why do we need this?

**Comment by marek-safar** [comment]
It makes csc faster

File: msbuild-mono-deploy.in [hunk 1]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec @bindir@/mono $MONO_OPTIONS @mono_instdir@/msbuild/15.0/bin/MSBuild.dll "$@"
+exec @bindir@/mono MONO_GC_PARAMS=nursery-size=64m $MONO_OPTIONS @mono_instdir@/msbuild/15.0/bin/MSBuild.dll "$@"
```
Comment (akoeplinger) [inline msbuild-mono-deploy.in:2]: @marek-safar this looks wrong, shouldn't the env var being set before `exec` ?

We should probably also make it `MONO_GC_PARAMS="nursery-size=64m,$MONO_GC_PARAMS"` so we don't overwrite existing values.

Comment (radical) [reply msbuild-mono-deploy.in:2]: yikes, you are right! I'll fix that, thanks!

Comment (marek-safar) [reply msbuild-mono-deploy.in:2]: I am not sure the doc says `exec env VARIABLE=value` but I agree with ,$MONO_GC_PARAMS

Comment (radical) [reply msbuild-mono-deploy.in:2]: It was anyway being used *after* `@bindir@/mono`, so it was incorrect anyway. https://github.com/mono/msbuild/commit/a2353ce6fbaa678e8a8f3f2ebbb1856a6e707d5e

**Pull request merged**